ansible_collections/arista/avd/molecule/evpn_underlay_ebgp_overlay_ebgp/documentation/devices/DC1-LEAF2B.md

```diff
@@ -483,7 +483,12 @@ interface Port-Channel7
    switchport
    switchport trunk allowed vlan 110-111,120-121,130-131,160-161
    switchport mode trunk
-   mlag 7
+   !
```

Comment:
this `!` should not be there. It is just another config line within the same section of the interface. A show running-config would not have this either.

Reply:
Resolved in last commit (d3852ae)